pages/linux/torsocks.md

```diff
@@ -0,0 +1,12 @@
+# torsocks
+
+> Use any application through the Tor network.
+> More information: <https://gitlab.torproject.org/tpo/core/torsocks/>.
+
+- Run a command using Tor:
+
+`torsocks {{command}}`
+
+- Enable or disable Tor in this shell:
+
+`. torsocks {{on/off}}`
```

Comment:
```suggestion
`. torsocks {{on|off}}`
```
We generally use this operator to highlight multiple options supported by the placeholder.